views/articles/new.ejs

```diff
@@ -0,0 +1,23 @@
+<% include ../partials/header.ejs %>
+
+<ul class="nav nav-tabs">
+  <li role="presentation"><a href="/">Home</a></li>
+  <li role="presentation"><a href="/contact">Contact</a></li>
+  <li role="presentation"><a href="/about">About</a></li>
+  <li role="presentation"><a href="/articles">Articles</a></li>
+  <li role="presentation" class="active"><a href="/articles/new">New</a></li>
+</ul>
+<div class="container">
+    <h1>New Article</h1>
+    <form action="/articles" method="POST">
+      <label>Title:</label>
+      <input type="text" name="name">
```

Comment:
For your articles to post correctly, we need to the name to be the key that you use in your articles object:
```
<input type="text" name="title">
<input type="text" name="body">
```